index.html

```diff
@@ -8,33 +8,41 @@
     <link rel="stylesheet" href="style.css">
     <script src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/4.1.1/crypto-js.min.js"></script>
 </head>
-
+<body>
 <div class="container">
     <div class="content">
         <header>
             <div class="search-bar">
                 <form id="searchForm">
-                    <input type="text" id="searchInput" placeholder="Search" required> 
+                    <input type="text" id="searchInput" placeholder="Search" required>
                     <button type="submit">Search</button>
                 </form>
 
             </div>
 
         </header>
 
+        <nav>
+
+        </nav>
+
         <main>
             <h1>Marvel's Heroes and Villains</h1>
-            <section id="characterInfo">
-                
+            <section id="characterInfo" class="hero-villain-container">
+                <div id="heroInfo" class="hero-side"></div>
             </section>
+
+            <nav>
+                <a href="comics.html" id="comicsLink"></a>
+            </nav>
```

Comment:
Look like you also have an empty `nav` on line 25. Maybe this one would be better placed there.

Reply:
Also, besides these 2 `nav`s, your JS adds another one dynamically. That's a bit confusing.